src/components/expression/expression-renderer.js

```diff
@@ -442,7 +522,7 @@ genotet.ExpressionRenderer.prototype.drawMatrixConditionLabels_ = function() {
   labels
     .attr('transform', genotet.utils.getTransform([
```

Comment:
When you want to apply a transform to every element under a group, apply to the group instead.